packages/wonder-blocks-button/components/button.md

```diff
@@ -432,3 +460,52 @@ const styles = StyleSheet.create({
     </View>
 </View>
 ```
+
+When an action is going to take a while, show a spinner during that time.
```

Comment:
That's a known issue.  Sorry for not documenting it.  It's pretty low priority though, but it should be tracked in JIRA.

Reply:
Oops, that's the effect of one of my changes (#191). If you revert that change, you get the behavior John described.

The reason behind #191 is because of the "Select all"/"Select none" options on the select dropdowns. What was happening was that "Select all" would become disabled once you clicked it, but then it wouldn't registered the `mouseleave` and would be highlighted once re-enabled, even if the mouse was no longer over it.

Perhaps a better solution would be to include the default `mouseenter`, `mouseleave`, `onblur` as part of the disabled handlers.

Reply:
> Perhaps a better solution would be to include the default mouseenter, mouseleave, onblur as part of the disabled handlers.

Yeah, that's probably the right way to go.